flake.nix

```
@@ -0,0 +1,65 @@
+{
+  description = "A list of useful scripts for cardano-node development";
+  inputs = {
+    systems.url = "github:nix-systems/default";
+    haskellNix.url = "github:input-output-hk/haskell.nix";
+    nixpkgs.follows = "haskellNix/nixpkgs-unstable";
+    utils.url = "github:numtide/flake-utils";
+    utils.inputs.systems.follows = "systems";
+  };
+
+  outputs = { nixpkgs, utils, haskellNix, ... }:
+    utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+          inherit (haskellNix) config;
```

```suggestion
    nixpkgs.follows = "haskellNix/nixpkgs-unstable";
    utils.url = "github:numtide/flake-utils";
    utils.inputs.systems.follows = "systems";
  };

  outputs = { nixpkgs, utils, ... }:
    utils.lib.eachDefaultSystem (system:
      let
        pkgs = import nixpkgs {
          inherit system;
```
It seems neither `config` nor `haskellNix` is being used, so it could be removed. That also reduces the size of `flake.lock` considerably. But if you think it may be used eventually, maybe it is easier to leave it there than bringing it back later. Just a thought

---

Right, I don't need haskellNix or config here. But the input is useful I think. This saves some disk space, because we're reusing nixpkgs version from haskell.nix.